modules/hcp-aws-create-role-sts-creds.adoc

```diff
@@ -14,10 +14,16 @@ Before creating a hosted cluster on {aws-first}, you must create an {aws-short}
 +
 [source,terminal]
 ----
-$ aws sts get-caller-identity --query "Arn"
+$ aws sts get-caller-identity --query "Arn" --output text
 ----
 +
-Note the ARN of your user to use in the next step.
+.Example output
+[source,terminal]
+----
+arn:aws:iam::1234567890:user/<aws_username>
+----
++
+You need the `arn` value from the output to use in the next step.
```

Comment:
The way this is written, I'm not sure if the "arn" value is just a piece of this output, or if it's the entire output, inserted into the <arn> replaceable value in the next step.

If it's the former then clarifying which piece to use would be helpful.

If it's the latter maybe something like the following would be clearer?

```suggestion
Use this output as the value for `<arn>` in the next step.
```

Reply:
Good suggestion!